_en/0001-01-01-membership.html

```diff
@@ -10,36 +10,39 @@
 <div class="row">
     <div class="col-md-8 col-md-offset-2 center-justified">
         <p>
-            javaBin is open for everyone who wants to be a member. Being a member with a paid membership means that you support
-            the Java community in Norway and helping us organize meetups and JavaZone.
+            javaBin is open for everyone who wants to become a member. Being a paying member means that you support
+            the Java community in Norway and help us organize various events throughout the year, and JavaZone.
```

Comment:
javaBin is open to anyone who wants to become a member. Every membership supports the Java community in Norway and helps us to organize various events throughout the year, including JavaZone.